blueprints/app/files/package.json

```diff
@@ -66,8 +66,8 @@
     "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0<% if (!embroider) { %>",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2<% } %>",
-    "ember-data": "~5.4.0-beta.12",
+    "ember-cli-terser": "^4.0.2<% } %>",<% if (emberData) { %>
```

Comment:
Template tags must be defined inside the quotes, so the auto-update script still works.
I made https://github.com/ember-cli/ember-cli/pull/10616 to help with this.

Reply:
fixed!